acceptance/README.md

```diff
@@ -0,0 +1,145 @@
+## Acceptance Tests
+
+The acceptance tests are blackbox* tests that are expected to interact with resources on a real GitHub instance.
```

Comment:
Unsure if we want to say more about `testscript` module as I find the documentation a little hard to follow personally, but pointing readers to the homepage to read about the language seems reasonable.  

```suggestion
The acceptance tests are blackbox* tests that are expected to interact with resources on a real GitHub instance.  They are built on top of [`go-internal/testscript` module](https://pkg.go.dev/github.com/rogpeppe/go-internal/testscript), which supports a variety of commands for defining filesystem-based tests.
```